js/src/tooltip.js

```diff
@@ -123,10 +122,6 @@ const TRIGGER_MANUAL = 'manual'
 
 class Tooltip {
   constructor(element, config) {
-    if (typeof Popper === 'undefined') {
```

Comment:
Shouldn't we keep this check?

Reply:
Popper.js doesn't export an object, so it'll force us to import everything to do that check, plus it's in our docs our tooltip/popover/dropdown need Popper.js and it'll throw an error